src/devtools/client/debugger/src/components/TestInfo/TestStepItem.tsx

```diff
@@ -44,96 +49,112 @@ export function TestStepItem({ argString, index, id }: TestStepItemProps) {
+            const cmdObject = cmdResult.data.objects?.find(
+              o => o.objectId === cmdResult.returned?.object
             );
-            const consoleProps = consolePropsPauseData.objects?.find(
-              o => o.objectId === consolePropsProperty.object
+            const length: number | undefined = cmdObject?.preview?.properties?.find(
+              o => o.name === "length"
+            )?.value;
+            const subjects = Array.from({ length: length || 0 }, (_, i) =>
+              cmdResult.data.objects?.find(
+                obj =>
+                  obj.objectId ===
+                  cmdObject?.preview?.properties?.find(p => p.name === String(i))?.object
+              )
             );
 
-            if (consoleProps?.preview) {
-              // suppress the prototype entry in the properties output
-              consoleProps.preview.prototypeId = undefined;
-            }
+            const nodeIds = subjects.filter(s => s?.preview?.node).map(s => s?.objectId!);
+            setSubjectNodePauseData({ pauseId: endPauseResult.pauseId, nodeIds });
+          }
+
+          if (messageEnd?.logVariable) {
+            const logResult = await client.evaluateExpression(
+              endPauseResult.pauseId,
+              messageEnd.logVariable,
+              callerFrame.frameId
+            );
 
-            setLocalPauseData({ pauseId: pauseResult.pauseId, consoleProps });
+            const consolePropsProperty = returnFirst(logResult.data.objects, o => {
+              return logResult.returned && o.objectId === logResult.returned.object
+                ? returnFirst(o.preview?.properties, p => (p.name === "consoleProps" ? p : null))
+                : null;
+            });
+
+            if (consolePropsProperty?.object) {
+              const consolePropsPauseData = await client.getObjectWithPreview(
+                consolePropsProperty.object,
+                endPauseResult.pauseId
+              );
+              const consoleProps = consolePropsPauseData.objects?.find(
+                o => o.objectId === consolePropsProperty.object
+              );
+
+              if (consoleProps?.preview) {
+                // suppress the prototype entry in the properties output
+                consoleProps.preview.prototypeId = undefined;
+              }
+            }
           }
+
+          setLocalPauseData({
+            startPauseId: startPauseResult?.pauseId,
+            endPauseId: endPauseResult.pauseId,
+            consoleProps,
+          });
         }
       } catch {
         setLocalPauseData(undefined);
       }
 
       return null;
     })();
-  }, [client, messageEnd, pointEnd, setConsoleProps, setPauseId]);
+
+    // return async () => {
+    //   if (endPauseResult) {
+    //     await client.releasePause(endPauseResult.pauseId);
+    //   }
+    //   if (startPauseResult) {
+    //     await client.releasePause(startPauseResult.pauseId);
+    //   }
+    // }
```

Comment:
There's a `Session.releasePause` command but it isn't supported on the client yet. Should it be? Should we be releasing pauses?